packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -286,12 +286,16 @@ export async function createProvisionalMemberships({
   try {
     await prisma.membership.createMany({
       data: invitees.flatMap((invitee) => {
+        const organizationRole = invitee?.teams?.[0]?.role;
         const data = [];
         // membership for the team
         data.push({
           teamId: input.teamId,
           userId: invitee.id,
-          role: input.role as MembershipRole,
+          role:
```

Comment:
this is to join the pending members as their respective org role

[Screencast from 10-12-23 02:10:03 PM IST.webm](https://github.com/calcom/cal.com/assets/32706411/d431e62c-8d44-4e32-9363-8353d49e5a44)